plugin.video.drnu/addon.xml

```diff
@@ -0,0 +1,41 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<addon id="plugin.video.drnu" version="5.0.13" name="DR TV" provider-name="TermeHansen">
+    <requires>
+        <import addon="xbmc.python" version="2.26.0"/>
+        <import addon="script.module.requests" version="2.22.0" />
+        <import addon="script.module.requests-cache" version="0.4.13"/>
+        <import addon="script.module.simplejson" version="2.0.10"/>
```

Comment:
please remove the simplejson dependency it is not needed anymore (this was only needed for helix)